### PR TITLE
Add logger level configuration support

### PR DESCRIPTION
**Pull request by siberianlove**
Based on issue: https://github.com/sameersbn/docker-redmine/issues/140

When my redmine log storage was low i was looking for log rotation solution for my redmine instance and discovered that it has one out of the box: https://github.com/search?q=repo%3Asameersbn%2Fdocker-redmine%20logrotate&type=code

It's just my instance generated so much info logs by default. So it'l be great if administrators can configure amount of generated logs but in this repo i didn't found any solution for configuring logger level so i implemented it.

Tested on my instance.

Thx for the great repo by the way! When i discovered this repo it became a lot easier to administrate redmine.

### CODE REVIEW

File: assets/runtime/functions [hunk 1]
```diff
@@ -329,13 +329,20 @@ redmine_configure_memcached() {
...
+      echo "config.action_controller.perform_caching  = true"
+      echo "config.cache_classes = true"
+      echo "config.cache_store = :mem_cache_store, ${MEMCACHE_HOST}:${MEMCACHE_PORT}"
+    } >> "${REDMINE_MEMCACHED_CONFIG}"
   fi
 }
 
+redmine_configure_logger() {
+  echo "Configuring redmine::logger..."
+  echo "config.log_level = :${LOGGER_LEVEL:-info}" >> "${REDMINE_MEMCACHED_CONFIG}"
```
Comment (jcormier) [inline assets/runtime/functions:343]: Also put the LOGGER_LEVEL default in env-defaults so its consistent with the other settings.

File: assets/runtime/functions [hunk 2]
```diff
@@ -329,13 +329,20 @@ redmine_configure_memcached() {
...
-    install_template ${REDMINE_USER}: redmine/additional_environment.rb ${REDMINE_MEMCACHED_CONFIG} 0644
-    update_template ${REDMINE_MEMCACHED_CONFIG} \
-      MEMCACHE_HOST \
-      MEMCACHE_PORT
+    {
+      echo "config.gem 'dalli'"
+      echo "config.action_controller.perform_caching  = true"
+      echo "config.cache_classes = true"
+      echo "config.cache_store = :mem_cache_store, ${MEMCACHE_HOST}:${MEMCACHE_PORT}"
+    } >> "${REDMINE_MEMCACHED_CONFIG}"
```
[the remark this answers is not in the document]
Comment (siberianlove) [reply assets/runtime/functions]: Done

File: assets/runtime/functions (hunk 1 above)
Comment (siberianlove) [reply assets/runtime/functions]: Done

**Comment by jcormier** [comment]
Thanks

File: assets/runtime/functions [hunk 3]
```diff
@@ -329,13 +329,20 @@ redmine_configure_memcached() {
...
     echo "Configuring redmine::memcached..."
-    install_template ${REDMINE_USER}: redmine/additional_environment.rb ${REDMINE_MEMCACHED_CONFIG} 0644
-    update_template ${REDMINE_MEMCACHED_CONFIG} \
-      MEMCACHE_HOST \
-      MEMCACHE_PORT
+    {
+      echo "config.gem 'dalli'"
+      echo "config.action_controller.perform_caching  = true"
+      echo "config.cache_classes = true"
+      echo "config.cache_store = :mem_cache_store, ${MEMCACHE_HOST}:${MEMCACHE_PORT}"
```
[the remark this answers is not in the document]
Comment (jcormier) [reply assets/runtime/functions:336]: pushed commit to master branch, can you test and let me know before I push a new tag

**Pull request merged**